templates/taxi/driver_list.html

```diff
@@ -5,6 +5,14 @@ <h1>Driver list</h1>
   {% if driver_list %}
     <ul>
       {% for driver in driver_list%}
+        {% if driver.id == user.id %}
+          <li>
+            <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
+            ({{ driver.first_name }} {{ driver.last_name }})
+            <span style="color: orange">(Me)</span>
+          </li>
+        {% endif %}
```

Comment:
The current logic will only display the current user and not any other drivers. The `if` condition should be adjusted so that it highlights the current user but still displays all drivers. Consider moving the 'Me' label logic inside the loop without using an `if` condition that skips rendering other drivers.